packages/resource-deployment/runtime-config/runtime-config.ci.json

```diff
@@ -0,0 +1,3 @@
+{
+    "profileName": "ci"
```

Comment:
>profileName [](start = 5, length = 11)

Profile means environment right? Can we call it that way?

Reply:
Sure. I used this one because this is what is being used in our release variables.